poker-pals/src/components/pokerComponents/PlayerSeat.vue

```diff
@@ -291,9 +291,9 @@ export default {
     components: {
         Clock,
     },
-    props: [
+    props: [ // 'active' was an old prop that got left in by accident
```

Comment:
since active is no longer in there, you might be ok to remove this message

Reply:
sure, i just wanted others to see it was removed intentionally